src/ClockPanel.tsx

```diff
@@ -157,18 +122,17 @@ class UnthemedClockPanel extends PureComponent<Props, State> {
     return formattedTimeLeft;
   }
 
-  getCountupText(): string {
-    const { now } = this.state;
-    const { countupSettings, timezone } = this.props.options;
+  function getCountupText(): string {
+    const { countupSettings, timezone } = props.options;
 
     if (!countupSettings.beginCountupTime) {
       return countupSettings.beginText;
     }
 
     const timePassed = moment.duration(
       moment(now).diff(
-        moment(this.props.replaceVariables(countupSettings.beginCountupTime)).utcOffset(
-          this.getTZ(timezone).format('Z'),
+        moment(props.replaceVariables(countupSettings.beginCountupTime)).utcOffset(
+          getMoment(timezone).format('Z'),
```

Comment:
nit: instead of referencing `prop.replaceVariables` what do you think about destructing it along side the other props at line 41?

Reply:
I think it is a matter of style and it makes no difference so I rather leave it as it is.